### PR TITLE
Adding base colors in OC scheme

### PR DESCRIPTION
**Pull request by felixheidecke**
Adding colors based on @wuenschedesign layout

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
Please adjust the variations thought out the whole system
Screenshot from 2019-04-03 16-02-26

**Comment by DeepDiver1975** [comment]
Screenshot from 2019-04-03 16-07-31

**Comment by DeepDiver1975** [comment]
Screenshot from 2019-04-03 16-08-11

**Comment by felixheidecke** [comment]
@wuenschedesign and I will meet on Monday for some coloring and design stuff. Will postpone this to next week.

File: src/elements/OcIcon.vue [hunk 1]
```diff
@@ -101,18 +101,6 @@ export default {
 			<oc-icon variation="inverted" name="info"/>
 			<oc-icon variation="inverted" name="account_circle"/>
 		</div>
-		<div class="uk-padding-small">
-			<oc-icon variation="primary" name="close"/>
```
Comment (DeepDiver1975) [inline src/elements/OcIcon.vue]: system?

File: src/elements/OcIcon.vue [hunk 2]
```diff
@@ -101,18 +101,6 @@ export default {
...
-			<oc-icon variation="primary" name="delete"/>
-			<oc-icon variation="primary" name="info"/>
-			<oc-icon variation="primary" name="account_circle"/>
-		</div>
-		<div class="uk-padding-small">
-			<oc-icon variation="secondary" name="close"/>
```
Comment (DeepDiver1975) [inline src/elements/OcIcon.vue]: file-type?

**Pull request merged**